Name: More Dialing Shortcuts
Version: 2.1.0-1
Author: The Resetter
Description: Add 8 and 10 Digits to Dialing Shortcuts in Phone Preferences
Index: /usr/palm/applications/com.palm.app.phoneprefs/app/models/SystemService.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.phoneprefs/app/models/SystemService.js
+++ /usr/palm/applications/com.palm.app.phoneprefs/app/models/SystemService.js
@@ -54,7 +54,7 @@
 }
 
 SystemService.getShortCutNumbers = function(callback) {
-	var keys = ['3DigitNumber', '4DigitNumber', '5DigitNumber', '6DigitNumber', '7DigitNumber'];
+	var keys = ['3DigitNumber', '4DigitNumber', '5DigitNumber', '6DigitNumber', '7DigitNumber', '8DigitNumber', '10DigitNumber'];
 	var request = new Mojo.Service.Request(SystemService.systemserviceIdentifier, {
 			method: 'getPreferences',
 			parameters: {'keys': keys},

Index: /usr/palm/applications/com.palm.app.phoneprefs/app/controllers/dialingshortcut-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.phoneprefs/app/controllers/dialingshortcut-assistant.js
+++ /usr/palm/applications/com.palm.app.phoneprefs/app/controllers/dialingshortcut-assistant.js
@@ -8,10 +8,12 @@
     setup: function(){
 		this.digitList = [{label:$L('3 digits'), value:"3DigitNumber"},
 		                  	{label:$L('4 digits'), value:"4DigitNumber"},
-							{label:$L('5 digits'), value:"5DigitNumber"},
-		 					{label:$L('6 digits'), value:"6DigitNumber"},
-							{label:$L('7 digits'), value:"7DigitNumber"}
-						 ];
+					{label:$L('5 digits'), value:"5DigitNumber"},
+ 					{label:$L('6 digits'), value:"6DigitNumber"},
+					{label:$L('7 digits'), value:"7DigitNumber"},
+					{label:$L('8 digits'), value:"8DigitNumber"},
+					{label:$L('10 digits'), value:"10DigitNumber"}
+				 ];
 		this.digitListModel = {
 				currentSelection: '4DigitNumber',
 				choices: this.digitList
Index: /usr/palm/applications/com.palm.app.phone/app/models/DialingPrefix.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.phone/app/models/DialingPrefix.js
+++ /usr/palm/applications/com.palm.app.phone/app/models/DialingPrefix.js
@@ -1,4 +1,4 @@
-//luna-send -n 1 palm://com.palm.systemservice/getPreferences '{"keys":["3DigitNumber", "4DigitNumber", "5DigitNumber", "6DigitNumber", "7DigitNumber"]}'
+//luna-send -n 1 palm://com.palm.systemservice/getPreferences '{"keys":["3DigitNumber", "4DigitNumber", "5DigitNumber", "6DigitNumber", "7DigitNumber", "8DigitNumber", "10DigitNumber"]}'
 DialingPrefix = function() {
 	var keys = ["3DigitNumber", "4DigitNumber", "5DigitNumber", "6DigitNumber", "7DigitNumber"];
 	
@@ -1,6 +1,6 @@
 //luna-send -n 1 palm://com.palm.systemservice/getPreferences '{"keys":["3DigitNumber", "4DigitNumber", "5DigitNumber", "6DigitNumber", "7DigitNumber"]}'
 DialingPrefix = function() {
-	var keys = ["3DigitNumber", "4DigitNumber", "5DigitNumber", "6DigitNumber", "7DigitNumber"];
+	var keys = ["3DigitNumber", "4DigitNumber", "5DigitNumber", "6DigitNumber", "7DigitNumber", "8DigitNumber", "10DigitNumber"];
 	
 	this.prefixes = {};
 	

